src/tests/multihost/alltests/conftest.py

```diff
@@ -222,11 +222,12 @@ def set_dslimits(session_multihost, request):
 
     def restore_dslimits():
         """ Restore the default sizelimit """
-        restore_size = [(ldap.MOD_REPLACE, 'nsslapd-sizelimit', [b'2000'])]
-        (ret, _) = ldap_inst.modify_ldap(configdn, mod_limit)
+        restore_size = [(ldap.MOD_REPLACE, 'nsslapd-lookthroughlimit',
```

Comment:
Is the s/sizelimit/lookthroughlimit/ change intentional?

Reply:
Disregard, I see why now :)

IIUC, we should change `restore_size` to `restore_lookthrough` now

Reply:
Done. Thanks for the review!